eval/src/vespa/eval/eval/tensor_function.cpp

```diff
@@ -143,6 +166,67 @@ Map::compile_self(Stash &) const
     return Instruction(op_tensor_map, to_param(_function));
 }
 
+vespalib::string
+name_of(map_fun_t fun)
```

Comment:
Consider have a common mapping from function to name to avoid duplication in dump_target.cpp.